app/api/helpers/tasks.py

```diff
@@ -190,8 +189,8 @@ def export_ical_task(self, event_id, temp=True):
             save_to_db(event)
 
     except Exception as e:
-        print(traceback.format_exc())
         result = {'__error': True, 'result': str(e)}
+        logging.warning('Error in ical download')
```

Comment:
It's not a handled exception unless we have some fallback to do in the block

Reply:
@ iamareebjamal Alright. Logging on appropriate levels done. Can you check if it's right?